src/services/services.ts

```diff
@@ -1772,15 +1772,24 @@ module ts {
         // Filename can be non-ts file.
         options.allowNonTsExtensions = true;
 
+        // We are not returning a lib file when asked, so pass this flag to
+        // avoid reporting a file not found error
+        options.noLib = true;

```

Comment:
not sure how to explain this more.. i updated the comment message, the reference here is to getSourceFile method in the host implementation.. see:
```ts
 getSourceFile: (fileName, target) => fileName === inputFileName ? sourceFile : undefined,
```